Cargo.lock

```diff
@@ -690,6 +712,15 @@ dependencies = [
  "windows-targets 0.48.5",
 ]
 
+[[package]]
+name = "parse-zoneinfo"
+version = "0.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f4ee19a3656dadae35a33467f9714f1228dd34766dbe49e10e656b5296867aea"
+dependencies = [
+ "regex 0.2.11",
```

Comment:
This is an ancient version of the regex crate -- is parse-zoneinfo even still maintained?

I don't think it would be good to bring such an ancient regex back into the rustc workspace.

Reply:
Ah the main issue is that this version of parse-zoneinfo is from 6 years ago. There's version 0.3.0 from 4 years ago that uses a more recent `regex`.

I don't think we should add a dependency on regex 0.2. That's just too old.